### PR TITLE
Update repoman config for new svc and subsvc labels.

### PR DESCRIPTION
**Pull request by BillWagner**
Update repo man config for new svc and subsvc labels.

Also, update the automation that was configured for the .NET 8 project to the .NET 9 project.

### CODE REVIEW

**Review by adegeo — APPROVED** [review]
Approved with one open question

File: .repoman.yml [hunk 1]
```diff
@@ -111,278 +111,278 @@ pull_request:
...
+                - labels-add: ["dotnet-visualbasic/svc", "errors-warnings/subsvc"]
 
   labeled:
 
-    # Add to .NET 6 project if .NET 6 label added
+    # Add to .NET 9 project if .NET 6 label added
     - check:
         - type: query
-          value: "length(Issue.labels[?name == ':checkered_flag: Release: .NET 6']) != `0`"
+          value: "length(Issue.labels[?name == ':checkered_flag: Release: .NET 9']) != `0`"
```
[the remark this answers is not in the document]
Comment (gewarren) [reply .repoman.yml:383]: Yes, for example, if I add the .NET 9 label to a breaking change issue.

[the remark this answers is not in the document]
Comment (BillWagner) [reply .repoman.yml:383]: Ignore previous comment. That's wrong.

I can add the workflow to "auto add item to project" whenever the .NET 9 label is added, the item is added to the .NET 9 project. I'll remove this entry

[the remark this answers is not in the document]
Comment (BillWagner) [reply .repoman.yml]: I did add it to the workflow on the .NET 9 project: https://github.com/orgs/dotnet/projects/362/workflows/15238388

(That link may not be publicly visible as it's in the project config)

**Pull request merged**